orchestra/static/orchestra/todos/todo-checklist.html

```diff
@@ -1,12 +1,19 @@
-<div class="list-by-status" ng-if="(todos | filter: {skipped_datetime: showSkipped?'!!':'!'}).length > 0">
-  <!-- <h4>{{title}}</h4> -->
-  <div class="todo"
-       ng-repeat="todo in todos | filter: {skipped_datetime: showSkipped?'!!':'!'}">
+<script type="text/ng-template" id="nodes_renderer.html">
+  <div ui-tree-handle class="tree-node tree-node-content">
     <label ng-class="{'text-danger': isInDanger(todo)}">
       <input type="checkbox"
              ng-hide="showSkipped"
              ng-model="todo.completed"
              ng-change="updateTodo(todo)">
+
+      <i aria-hidden="true" ng-if="todo.items && todo.items.length > 0" data-nodrag
+        class="fa"
+        ng-class="{
+          'fa-caret-right': collapsed,
+          'fa-caret-down': !collapsed
+        }"></i>
+      <i class="fa fa-circle-o" aria-hidden="true" ng-if="(!todo.items || todo.items.length === 0) && showSkipped" data-nodrag></i>
+      <span ng-hide="!todo.items || todo.items.length < 1">[{{ ( todo.items | filter: { completed: true } ).length }}/{{ todo.items.length }}] </span>
       <span class="todo__role">{{steps[taskSlugs[todo.task]].name}}:&nbsp;</span>
```

Comment:
Should we consider render the sub-todo in lighter font-weight? Will there be a limit on how deep the tree is?

Reply:
Although `angular-ui-tree` allows us to enforce depth of a tree, I am not forcing any depth restrictions currently.